$ make clean all WITHOUT=ENCHANT
[...]
ld: error: undefined symbol: spelling_enchant_provider_new
>>> referenced by spelling-provider.c
>>>               libspelling-1.a.p/spelling-provider.c.o:(spelling_provider_get_default) in archive lib/libspelling-1.a

--- meson.build.orig	2024-03-04 00:39:28 UTC
+++ meson.build
@@ -13,7 +13,7 @@ config_h.set_quoted('PACKAGE_LOCALE_DIR', join_paths(g
 config_h.set_quoted('PACKAGE_VERSION', meson.project_version())
 config_h.set_quoted('GETTEXT_PACKAGE', 'libspelling')
 config_h.set_quoted('PACKAGE_LOCALE_DIR', join_paths(get_option('prefix'), get_option('datadir'), 'locale'))
-config_h.set10('HAVE_ENCHANT', get_option('enchant').enabled())
+config_h.set('HAVE_ENCHANT', get_option('enchant').enabled())
 
 # Detect and set symbol visibility
 if get_option('default_library') != 'static'
